Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithBypassDamper.mo

```diff
@@ -0,0 +1,296 @@
+  connect(hex.port_b1, port_b1)
+    annotation (Line(points={{10,16},{58,16},{58,80},{100,80}},
+    color={0,127,255}));
+  connect(hex.port_b2, VExh_flow.port_a)
+    annotation (Line(points={{-10,4},{-22,4},{-22,-20},{-42,-20}},
+    color={0,127,255}));
+  connect(VExh_flow.port_b, port_b2)
+    annotation (Line(points={{-56,-20},{-70,-20},
+    {-70,-60},{-180,-60}}, color={0,127,255}));
+  connect(hex.port_a2, damExh.port_b)
+    annotation (Line(points={{10,4},{60,4},{60,-30}}, color={0,127,255}));
+  connect(VExh_flow.V_flow, effCal.VExh_flow)
+    annotation (Line(points={{-49,-11.2},{-49,60},{-120,60},{-120,14},{-102,14}},
+    color={0,0,127}));
+  annotation (
+        defaultComponentName="whe",
+        Icon(coordinateSystem(extent={{-100,-100},{100,100}}),
+        graphics={
+        Rectangle(
+          extent={{32,-56},{94,-64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{32,64},{94,56}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-92,-55},{-30,-64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-94,65},{-32,56}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(extent={{6,88},{38,-90}},
+          lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{0,100},{0,100}},
+          lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.None),
+        Rectangle(
+          extent={{-2,88},{22,-98}},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Ellipse(
+          extent={{-38,88},{-4,-90}},
+          lineColor={28,108,200},
+          fillColor={28,108,200},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-22,-90},{22,-90}}, color={28,108,200}),
+        Line(points={{-20,88},{22,88}}, color={28,108,200}),
+        Line(points={{-58,64},{-58,96},{64,96},{64,64}}, color={28,108,200}),
+        Line(points={{-58,-64},{-58,-96},{64,-96},{64,-64}}, color={28,108,200}),
+        Text(
+          extent={{-147,-104},{153,-144}},
+          textColor={0,0,255},
+          textString="%name")}),
+          Diagram(
+        coordinateSystem(preserveAspectRatio=true, extent={{-180,-100},{100,140}})),
+Documentation(info="<html>
+<p>
+Model of a generic, sensible, and latent air-to-air heat recovery wheel, that consists of 
+a heat exchanger and supply/exhaust airflow bypass dampers.
```

Comment:
`Model of a generic, sensible and latent air-to-air heat recovery wheel. It consists of a heat exchanger and two dampers to bypass the supply and exhaust airflow. The operation of the heat recovery wheel is controlled through adjusting the bypassing supply and exhaust airflow.`